packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -53,6 +53,11 @@ const propTypes = {
    * Name of the date input. The name should be unique.
    */
   name: PropTypes.string.isRequired,
+  /**
+   * A callback function triggered when the entire date time picker component loses focus.
+   * This event does not get triggered when the focus is moved from the date input to the time input because the focus is still within the main date time picker component.
+   */
+  onBlur: PropTypes.func,
```

Comment:
Should we allow consumers to pass in individual onDateBlur and onTimeBlur callbacks as well?

Reply:
Do you know of a use case where consumers would need the individual onDateBlur and onTimeBlur? I'm not against providing these more granular onBlurs. Perhaps, add them when the need arises.

Reply:
My preference would be to add as a use-case arises.